revscoring/datasources/site.py

```diff
@@ -0,0 +1,32 @@
+import mw
+
+from .datasource import Datasource
+
+
+def process_doc(session):
+    
+        doc = session.site_info.query(properties={'general', 'namespaces',
+                                                  'namespacealiases'})
+        return doc
+
+doc = Datasource("site.doc", process_doc, depends_on=['session'])
+
+
+def process_namespace_map(site_info_doc):
+    
+    aliases = site_info_doc.get('namespacealiases', [])
+    alias_map = {}
```

Comment:
This fixes https://github.com/halfak/Revision-Scoring/commit/2d6b636b16b7760a5f904c3b83b835251d638b8b#diff-288091a7883dc67d96ad359400b9ac0dR10

Thanks! :-)